src/app/file/psd_format.cpp

```diff
@@ -364,16 +367,31 @@ class PsdDecoderDelegate : public psd::DecoderDelegate {
       const int timeMs = frameInfo.duration * 10;
       m_sprite->setFrameDuration(frame_t(i), timeMs);
     }
-    // at this point, we're supposed to be able to set the activeFrame
+
+    auto& slices = m_sprite->slices();
+    for (const auto& slice : m_slices.slices) {
```

Comment:
Couldn't we do this just in `onSlicesData()` without storing the slices in `m_slices`?